assets/index.scss

```diff
@@ -400,7 +401,7 @@ article.list {
 }
 
 .wire-column__preview--open {
-    width: 440px;
+    width: 480px;
```

Comment:
Hi @aleksandarjelicic, I increased the preview panel size to 480 as the date and the action buttons were not fitting properly. Feel free to change it if it creates an issue.